ThirdPartyNotices

```diff
@@ -7,9 +7,9 @@ see the [LICENSE](LICENSE) file, and grant you a license to any code in the repo
 Microsoft, Windows, Microsoft Azure and/or other Microsoft products and services referenced in the documentation
 may be either trademarks or registered trademarks of Microsoft in the United States and/or other countries.
 The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
-Microsoft's general trademark guidelines can be found at https://go.microsoft.com/fwlink/?LinkID=254653.
+You can find Microsoft general trademark guidelines at [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks).
```

Comment:
- Add spacing.
- Add spacing.
- Add spacing.  
- Add spacing.
- Drop locale in URL.

```suggestion

You can find Microsoft general trademark guidelines at [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/legal/intellectualproperty/trademarks).
```

Reply:
This is standard text from CELA that that appears in many repos so I don't think it should be customized. I just made the changes CELA requested per the linked bug.